### ISSUE TITLE
Some game data files may have been corrupted

### ISSUE THREAD

**Issue by dmc5179** [criticalstack/quake-kube#15]
After deploying example.yaml on my cluster I can see in the pod server logs:

```
**************************************************
WARNING: baseq3/pak0.pk3 is present but its checksum (2985612116)
is not correct. Please re-copy pak0.pk3 from your
legitimate Q3 CDROM.
**************************************************
```

Not sure if this a known non-issue or something that needs to be re-copied into the container image.

**Comment by ChrisRx** [comment]
I may be misunderstanding the need to use the standalone game build.  I haven't tried to use one of the custom quake games yet, but wanted to ensure that it could be done in the future, however, it seems like just setting the basegame should be all that is necessary for that.  I think some more investigation is necessary here, my biggest concerns would be that it somehow violates the license (although looking at the q3 demo license it doesn't appear to).

**Comment by ChrisRx** [comment]
I think this should be addressed with #16 but I welcome any follow-up discussion (we can reopen this or start a new issue). Thanks again!
